.travis.yml

```diff
@@ -3,11 +3,11 @@
 sudo: false
 language: node_js
 node_js:
+  - '6'
   - '8'
-  - '9'
   - '10'
-  - '11'
   - '12'
+  - 'node'
```

Comment:
Why this line ? to test on current stable release ? it should be `'stable'` in this case

Reply:
`stable` and `node` are the same for TravisCI afaik

Reply:
```suggestion
  - 'stable'
```
So I suggest to use `'stable'` because it’s more explicite.